q.cpp

```diff
@@ -0,0 +1,245 @@
+#include <stdio.h>
+#include <string.h>
+#include <malloc.h>
```

Comment:
I didn't know... During years I have though that `make` always need a `makefile` file...  Vivir para ver :)

NTC